deploy/osd-route-monitor-operator/100-rmo.RouteMonitor.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: monitoring.openshift.io/v1alpha1
+kind: RouteMonitor
+metadata:
+  name: console
+spec:
+  route:
+    name: console
+    namespace: openshift-console
+  slo:
+    targetAvailabilityPercent: "99.5"
```

Comment:
This needs to be a string?

Reply:
There are no floating points support in the apiserver :(

Reply:
The types defined in the CRD? 
https://github.com/openshift/route-monitor-operator/blob/master/config/crd/bases/monitoring.openshift.io_routemonitors.yaml#L55  

We could update this to be a `number` as per: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#type

Reply:
thanks @dofinn ! I'll see how we do that in json stuffz

Reply:
@dofinn seems this is not 100% supported by operator-sdk / kubebuilder as of now
see https://github.com/kubernetes-sigs/controller-tools/issues/245 as people work on shying away from floats

also in an internal discussion I tried to understand how to implement doubles in the api resource, but didn't revieve any ❤️  on it (aside of @dastergon)

Reply:
ooo thats interesting. nice find ! @georgettica